src/subHeader.css

```diff
@@ -0,0 +1,77 @@
+
+/* sub header css start */
+.appSubHeader {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 50px 0px;
+  }
+  
+  .headerBold {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 60px 10px 10px 10px;
+    font-weight: 600 !important;
+  }
+  .headerHeadLine {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 80px;
+    line-height: 88px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #FFFFFF;
+  }
+  
+  .headerNormal {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 25px 0px;
+  }
+  
+  .headerText {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 30px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #FFFFFF;
+  }
+  
+  .headerButton {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0px;
+    gap: 35px;
+    justify-content: center;
+  }
+  .button {
+    padding: 10px 40px;
+    width: 178px;
+    height: 65px;
```

Comment:
Try to avoid setting `height` on elements.
unless it's necessary

use `padding`
by default the `font-size + padding === height`, ie the contents of the element